### PR TITLE
Qahook

### PR DESCRIPTION
**Pull request by bindumadhav**

### CODE REVIEW

File: BasicPlaybackSampleApp/BasicPlaybackSampleApp/Utils/SampleAppPlayerViewController.h [hunk 1]
```diff
@@ -21,10 +21,14 @@
 @property (weak, nonatomic) IBOutlet UISwitch *switch2;
 @property (weak, nonatomic) IBOutlet UITextField *text1;
 @property (weak, nonatomic) IBOutlet UITextField *text2;
+@property (strong, nonatomic) IBOutlet UIStackView *stackView1;
+@property (weak, nonatomic) IBOutlet UIView *playerView1;
+@property (weak, nonatomic) IBOutlet UITextView *textView;
 
 @property (strong, nonatomic) PlayerSelectionOption *playerSelectionOption;
+@property BOOL qaModeEnabled;
```
Comment (brol1dev) [inline BasicPlaybackSampleApp/BasicPlaybackSampleApp/Utils/SampleAppPlayerViewController.h]: set this attribute as nonatomic

File: BasicPlaybackSampleApp/BasicPlaybackSampleApp/Players/BasicSimplePlayerViewController.m [hunk 2]
```diff
@@ -81,6 +86,19 @@ - (void) notificationHandler:(NSNotification*) notification {
         [notification name],
         [OOOoyalaPlayer playerStateToString:[self.ooyalaPlayerViewController.player state]],
         [self.ooyalaPlayerViewController.player playheadTime], appDel.count);
+  
+  //In QA Mode , adding notifications to the TextView
+  if(self.qaModeEnabled==YES) {
+  NSString *message = [NSString stringWithFormat:@"Notification Received: %@. state: %@. playhead: %f count: %d",
```
Comment (brol1dev) [inline BasicPlaybackSampleApp/BasicPlaybackSampleApp/Players/BasicSimplePlayerViewController.m]: Define this `NSString *message = ...;` variable above the `NSLog`, as it is identical. Use `message` variable where needed.

File: BasicPlaybackSampleApp/BasicPlaybackSampleApp/Lists/BasicPlaybackListViewController.m [hunk 3]
```diff
@@ -16,6 +16,7 @@ @interface BasicPlaybackListViewController ()
 @property NSMutableArray *options;
```
Comment (brol1dev) [inline BasicPlaybackSampleApp/BasicPlaybackSampleApp/Lists/BasicPlaybackListViewController.m]: Could you set the nonatomic attribute to all properties that don't have it?

**Comment by bindumadhav** [comment]
sure will make the changes suggested

**Comment by bindumadhav** [comment]
@rcvrgs Made changes based on CR comments

**Comment by brol1dev** [comment]
Thanks for the changes :) LGTM

**Pull request merged**